src/Microsoft.Health.Fhir.Web/appsettings.json

```diff
@@ -56,6 +56,7 @@
       "MaxWaitTimeInSeconds": 5
     }
   },
+  "DataStore": "CosmosDb",
```

Comment:
nit: should this be under FhirServer since it's setting the data store for the FHIR server?

Reply:
I don't think so since this is a setting only for our web project and not the actual fhir server code.